CMakeLists.txt

```diff
@@ -90,6 +90,15 @@ if(TARGET Boost::program_options)
     target_link_libraries(sphinxsys_core INTERFACE Boost::program_options)
 endif()
 
```

Comment:
add option
option(SPHINXSYS_USE_SYCL "Build using SYCL acceleration or not" OFF)